GoogleForCreators/web-stories-wp
# PPC: Priority Issue Cards

## Context

Part of 2.0 PPC for 1.9 release. 

## Summary

Adding some skins to a handful of issues for the new 2.0 version of the prepublish checklist. 

## Relevant Technical Choices

Used constants where I could. 
Story level tests need to destructure the `story` in `useStory` so wired that up. 
Fix linting complaint about card styles. 

## To-do

More cards 

## User-facing changes

None, behind feature flag. 

## Testing Instructions

If you enable the new checklist experiment and open the checklist you can now see a handful of cards for the following issues (if you trip the issue). Panels are getting implemented separately. 

- storyTitle
- storyPosterPortraitSize
- storyPosterAspectRatio
- publisherLogoSize
- 
<img width="340" alt="Screen Shot 2021-06-22 at 2 05 56 PM" src="https://user-images.githubusercontent.com/10720454/122999897-ac3f8880-d363-11eb-8e71-5eb2b73332ad.png">


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #7920
